docs/source/markdown/podman-volume-exists.1.md

```diff
@@ -0,0 +1,43 @@
+% podman-volume-exists(1)
+
+## NAME
+podman\-volume\-exists - Check if the given volume exists
+
+## SYNOPSIS
+**podman volume exists** *volume*
+
+## DESCRIPTION
+**podman volume exists** checks if a volume exists. Podman will return an exit code
+of `0` when the volume is found. A `1` will be returned otherwise. An exit code of
+`125` indicates there was an other issue.
```

Comment:
'another' is one word. (This is not worth a re-push. I mention it only if you need to resubmit for another reason).

Reply:
U would rather fix this, since it is Human facing.
@Luap99 Fix this and then LGTM and we can merge.